datahub-dao/src/main/java/com/linkedin/datahub/util/CorpUserUtil.java

```diff
@@ -11,6 +11,9 @@
 
 
 public class CorpUserUtil {
+
+    private static final String DEFAULT_AVATAR = "http://localhost:8081/assets/images/default_avatar.png";
```

Comment:
What I meant is that you can drop `http://localhost:8081` prefix and let the browser figure that out automatically.

Reply:
See https://www.w3schools.com/html/html_filepaths.asp

Reply:
I think `datahub-web` has already this default and you might not even need to set this explicitly from mid-tier.

Reply:
@keremsahin1  I tried but it doesn't load the default avatar, particularly in the Ownership tab.
@mars-lan  I will try your suggestion, thanks for the link.

Reply:
Sure